src/site/content/en/handbook/web-dev-components/index.md

```diff
@@ -5,6 +5,8 @@ date: 2019-06-26
 updated: 2020-07-17
 description: |
   Learn how to use web.dev's UI and content components.
+feedback:
```

Comment:
At first glance I thought this was going to enable all of the default questions, but I think it's actually enabling all of the _conditional_ feedback questions.

You might rename it to something like `additional_feedback_questions`.

Reply:
In the spirit of YAGNI I think we can get rid of the option of enabling all conditional questions. When we need to enable conditional questions, we'll just do it one-by-one. If we add other conditional questions, I expect that they'll be for dramatically different purposes. E.g. some will be related to APIs, others may be along the lines of "Are you planning on implementing this feature?", other ones might be about browser compatibility. So I don't think it's likely that I would want to enable them all.

Reply:
Also, I think we can remove this test page because I just pushed a commit that adds the feature to a real page (the change password URL post)

Reply:
Ok, I removed the `all` flag and the `feedback` front matter from the handbook, but @robdodson @kaycebasques should we rename the property from `feedback` to `additional_feedback_questions`?